drivers/mipi_dbi/mipi_dbi_smartbond.c

```diff
@@ -0,0 +1,610 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT  renesas_smartbond_mipi_dbi
+
+#include <zephyr/drivers/mipi_dbi.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/irq.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/pm/device_runtime.h>
+#include <DA1469xAB.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/smartbond_clock_control.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/drivers/spi.h>
+#include <da1469x_lcdc.h>
+#include <da1469x_pd.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(smartbond_mipi_dbi, CONFIG_MIPI_DBI_LOG_LEVEL);
+
+#define SMARTBOND_IRQN		DT_INST_IRQN(0)
```

Comment:
Generally speaking, I'd prefer that this driver be multi-instance, unless there is a good reason not to make it support multiple device instances. These properties should likely be part of the `dev->config` structure